core/services.go

```diff
@@ -958,7 +896,7 @@ func (app *Application) synchronizeAuthman(clientID string, checkThreshold bool)
 
 	if len(authmanGroups) > 0 {
 		for _, authmanGroup := range authmanGroups {
-			err := app.synchronizeAuthmanGroup(clientID, &authmanGroup)
+			err := app.synchronizeAuthmanGroup(clientID, authmanGroup.ID)
```

Comment:
We should probably rewrite this whole `synchronizeAuthman` function since it is currently putting the admins in as  embedded members.  It would be better if it just directly updated the group memberships as needed or passed the admin data to `synchronizeAuthmanGroup` to be handled there.

Reply:
Thanks @mdryankov, I think we should make this change before merging this PR since I think our goal should be to have the `members` field remain empty after the migration once this is merged. Let me know if you would like to take a pass at re-writing this, or if you would like me to give it a shot. Thanks again!

Reply:
@shurwit I think the members within the group model are used only for migration purpose. Please resolve this entry if it's fine.